response.go

```
@@ -0,0 +1,44 @@
+}
+
+func (r BaseResponse) SetCode(code int) {
+	r.Code = code
+}
+
+func (r BaseResponse) SetContent(content any) {
+	r.Content = content
+}
+
+func (r BaseResponse) SetContentType(contentType string) {
+	r.ContentType = contentType
```

golangci-lint (1.64.8)

unusedwrite: unused write to field ContentType

(govet)